CONTRIBUTORS.md

```diff
@@ -74,6 +74,10 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
    * [Guillaume Beraudo](https://github.com/gberaudo)
 * [EndPoint](https://www.endpoint.com/)
    * [Dmitry Kiselev](https://github.com/kiselev-dv)
+* [Safe Software](https://www.safe.com)
```

Comment:
Yeah it was intentional. Should `3d-tiles` only contributors stay in that branch for now?